src/command_modules/azure-cli-sf/setup.py

```diff
@@ -34,7 +34,7 @@
 DEPENDENCIES = [
     'azure-servicefabric==5.6.130',
     'azure-cli-core',
-    'adal==0.4.3'
+    'adal>=0.4.3'
```

Comment:
core and profile packages use `adal>=0.4.3` so guessing it was copied from there.
Would like it consistent with the others so this looked fine to me.